Gemfile

```diff
@@ -33,12 +33,18 @@ gem 'faraday'
 # Keep information secure
 gem 'figaro'
 
+# Restful routes serialzer
+gem 'fast_jsonapi'
+
 # Create graphQL endpoints
 gem 'graphql'
 
-# Add wrapper for mixcloud api 
+# Add wrapper for mixcloud api
 gem 'mixcloud'
 
+# Allows easier use of AWS S3 bucket interactions
+gem 'aws-sdk-s3', '~> 1'
+
 group :development, :test do
```

Comment:
Do we need a travis gem?

Reply:
I think the Travis gem really just gives us a CLI, but it probably wouldn't hurt to include it